docs/UserGuide.md

```diff
@@ -90,18 +90,34 @@ gradlew run -Dargs="-view output_path/reposense-report"
 ```
 `-Dargs="..."` uses the same argument format as mentioned above.
 
-## CSV Config File
-The `CSV Config File` contains the list of repositories, and the corresponding target authors to track contribution of.
+## CSV Config Files
+The `repo-config.csv` contains the location and the branch of repositories to analyze.
```

Comment:
#### Repository configuration file
to configure the list of repositories to analyze and the respective repository level options. 
[repo-config.csv](../repo-config.csv) is an example of a configuration file setup. It should contain the following columns: